### ISSUE TITLE
Program not starting with windows 

### ISSUE THREAD

**Issue by Fascinated-Viewer** [Rem0o/FanControl.Releases#1459]
Every time I lunch windows, the program does not load with windows I have tried to troubleshoot this but have had no luck 

[log.txt](https://github.com/Rem0o/FanControl.Releases/files/10418294/log.txt)

Ryzen 1700
B450M Gaming by gigabyte 
GT 1030
32 Gb memory running at 3600
950watt PSU

**Comment by Rem0o** [comment]
Things to try/consider:

Did you extract the program properly in a folder?
Uncheck and check the start with windows.
What delay do you have set for start with windows?

**Comment by Fascinated-Viewer** [comment]
Yes the program has been extracted to a harddrive thats not the operating system
The program does not show up in start up with windows 
![Screenshot 2023-01-14 172830](https://user-images.githubusercontent.com/122690765/212486959-0bdcee64-ca82-49de-9560-0e2dcfb26e05.png)
![Screenshot 2023-01-14 173104](https://user-images.githubusercontent.com/122690765/212486976-7a6352ea-33ed-489b-9d59-9d53af105cab.png)

**Comment by Rem0o** [comment]
In won't appear in that list.   It's a scheduled task in the task scheduler.

**Comment by Fascinated-Viewer** [comment]
I`ve had a look at the task scheduler adn to me it looks alright unless theres something am missing...?
![Screenshot 2023-01-14 173851](https://user-images.githubusercontent.com/122690765/212487370-fa27e25e-3a90-4ec2-a8ee-411948a92563.png)

**Comment by Rem0o** [comment]
Try and right click run it, see if it makes FanControl starts.

**Comment by Fascinated-Viewer** [comment]
So the odd thing is that it was already running in the back ground and Clicked run and it ran the program fine
